src/websockets.js

```diff
@@ -387,10 +383,12 @@ try {
 }
 
 if (typeof process.env.WS_IPC_PATH === "string") {
-  Websockets.startIPC().catch(err => {
+  try {
+    Websockets.startIPC();
```

Comment:
This won't work because you're missing `await`; revert this entire change, `.catch` was fine